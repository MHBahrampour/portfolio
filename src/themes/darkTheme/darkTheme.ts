import { type ThemeOptions } from "@mui/material";

export const darkTheme: ThemeOptions = {
  palette: {
    mode: "dark",
    common: {
      black: "#0f1b2c",
      white: "#f2f2f2",
    },
    primary: {
      main: "#f2f2f2",
      light: "#f4f4f4",
      dark: "#a9a9a9",
      contrastText: "#0f1b2c",
    },
    secondary: {
      main: "#00a19d",
      light: "#33b3b0",
      dark: "#00706d",
      contrastText: "f2f2f2",
    },
  },
  shape: {
    borderRadius: 8,
  },
  typography: {
    allVariants: {
      fontFamily: "Poppins, sans-serif",
      color: "#f2f2f2",
    },
  },
};

// MUI Default Theme: https://mui.com/material-ui/customization/default-theme/
