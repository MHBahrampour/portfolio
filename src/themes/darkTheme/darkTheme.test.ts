import { darkTheme } from "./darkTheme";

it("checks darkTheme's snapshot", () => {
  expect(darkTheme).toMatchSnapshot();
});
