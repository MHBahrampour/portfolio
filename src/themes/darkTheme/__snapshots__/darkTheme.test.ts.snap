// Vitest Snapshot v1

exports[`checks darkTheme's snapshot 1`] = `
{
  "palette": {
    "mode": "dark",
  },
  "shape": {
    "borderRadius": 8,
  },
  "typography": {
    "fontFamily": "Poppins, sans-serif",
  },
}
`;
